regcomp.c

```diff
@@ -12828,13 +12831,15 @@ S_regpiece(pTHX_ RExC_state_t *pRExC_state, I32 *flagp, U32 depth)
         MARK_NAUGHTY_EXP(1, 4);     /* compound interest */
     }
 
+    /* Finish up the {m,n} case */
```

Comment:
This is rather the CURLY/CURLYX cases, since it also handles `?` and non-SIMPLE `+` and `*`.

Reply:
ok